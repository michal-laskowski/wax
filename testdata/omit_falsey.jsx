export function omit_falsey() {
    return (<>
        <in-inner>
            <sample-01>{null}</sample-01>
            <sample-02>{undefined}</sample-02>
            <sample-03>{false}</sample-03>
            <sample-04>{0}</sample-04>
            <sample-99>{null}|{undefined}|{false}|{0}</sample-99>
        </in-inner>
        <in-attr>
            <sample-01 v={null} />
            <sample-02 v={undefined}  />
            <sample-03 v={false}  />
            <sample-04 v={0} />
            <sample-99 v1={null} v2={undefined} v3={false} v4={0} />
        </in-attr>
    </>)
}