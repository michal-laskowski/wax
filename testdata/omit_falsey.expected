<in-inner>
    <sample-01></sample-01>
    <sample-02></sample-02>
    <sample-03></sample-03>
    <sample-04>0</sample-04>
    <sample-99>|||0</sample-99>
</in-inner>
<in-attr>
    <sample-01></sample-01>
    <sample-02></sample-02>
    <sample-03></sample-03>
    <sample-04 v="0"></sample-04>
    <sample-99 v4="0"></sample-99>
</in-attr>
